pdi-execution-engine/api/src/main/java/org/pentaho/di/engine/api/IExecutableOperation.java

```diff
@@ -0,0 +1,36 @@
+package org.pentaho.di.engine.api;
+
+/**
+ * ICallableOperation is the "materialized" version of
+ * an IOperation.  It represents the mapping of the
+ * structural specification of the op to it's concrete
+ * executable form, as applicable to the IEngine in which
+ * it is running.
+ *
+ * An ICallableOperation encapsulates a function capable of transforming input
+ * tuples received "From" parent operations and published
+ * "To" child ops.
+ */
+import rx.Subscriber;
+
+public interface IExecutableOperation extends IOperation {
+
+  void start();
```

Comment:
I threw .start() on as an afterthought when I was figuring out how the whole process kicks off.   Subscription::request seems like a better option.